packages/picasso/src/Badge/styles.ts

```diff
@@ -1,46 +1,45 @@
 import { rem } from '@toptal/picasso-shared'
-import { Theme, createStyles } from '@material-ui/core/styles'
+import { Theme, createStyles, StyleRules } from '@material-ui/core/styles'
 
-const getSizeProps = (unitInRem: string) => ({
+const getSizeProps = (unitInRem: string): StyleRules[keyof {}] => ({
   borderRadius: unitInRem,
   height: unitInRem,
-  lineHeight: unitInRem,
   minWidth: unitInRem
 })
 
 export default ({ palette }: Theme) =>
   createStyles({
     root: {
-      borderStyle: 'solid',
       borderWidth: '1px',
-      display: 'inline-flex',
+      borderStyle: 'solid',
       fontSize: rem('10px'),
-      fontWeight: 600,
-      alignItems: 'center',
-      justifyContent: 'center'
+      lineHeight: rem('12px'),
+      padding: `0 ${rem('1px')}`,
```

Comment:
```suggestion
      padding: `0 1px`,
```
`rem` utility can end up in some float number or some rounding

Reply:
done